pkg/reporting/cookbooks.go

```diff
@@ -96,7 +101,18 @@ func (r *CookbookRecord) NumCorrectable() int {
 	return i
 }
 
-func NewCookbooks(cbi CookbookInterface, searcher SearchInterface, runCookstyle, onlyUnused bool, workers int) (*CookbooksStatus, error) {
+func NewCookbooks(
+	cbi CookbookInterface, searcher SearchInterface,
+	runCookstyle, onlyUnused bool, workers int,
+) (*CookbooksStatus, error) {
+	wsDir, err := config.ChefWorkstationDir()
+	if err != nil {
+		return nil, err
+	}
+
+	cookbooksDir := filepath.Join(wsDir, analyzeCacheDir, analyzeCookbooksDir)
+	//debug("%s: using cookbooks directory %s", time.Now(), cookbooksDir)
```

Comment:
Commented out debug statement - I'm fine with leaving it if you did this intentionally

Reply:
I did, I have a bunch that I wanna tackle sooner rather than later. 🤞